ts/components/dialog/DeleteAccountModal.tsx

```diff
@@ -57,6 +59,16 @@ async function deleteEverythingAndNetworkData() {
     // DELETE EVERYTHING ON NETWORK, AND THEN STUFF LOCALLY STORED
     // a bit of duplicate code below, but it's easier to follow every case like that (helped with returns)
 
+    // clear all sogs inboxes (includes message requests)
+    const allRoomInfos = await getAllValidOpenGroupV2ConversationRoomInfos();
+    if (allRoomInfos && allRoomInfos.size > 0) {
+      // clear each inbox per sogs
+      for (const roomInfo of allRoomInfos.values()) {
+        // TODO CONTINUE testing - use a dummy account with some message requests and then if we restore from seed there should be no message requests.
+        await clearInbox(roomInfo);
```

Comment:
should probably  catch any errors here so we still continue deleting the DB if just one of the inbox fails?
like what if a sogs is offline, you still want to be able to remove your account.

Reply:
Good point. Does this make sense?

```ts
    // clear all sogs inboxes (includes message requests)
    const allRoomInfos = await getAllValidOpenGroupV2ConversationRoomInfos();
    if (allRoomInfos && allRoomInfos.size > 0) {
      // clear each inbox per sogs
      for (const roomInfo of allRoomInfos.values()) {
        // TODO CONTINUE testing - use a dummy account with some message requests and then if we restore from seed there should be no message requests.
        try {
          await clearInbox(roomInfo);
        } catch (error) {
          window.log.info(
            `[deleteEverythingAndNetworkData]: Failed to clear inbox ${roomInfo}`,
            error
          );
          continue;
        }
      }
    }
```

Reply:
yep that would work. I saw you updated the code with it already so closing this item.